newdle/client/src/components/answer/Option.js

```diff
@@ -25,6 +28,8 @@ Option.propTypes = {
   className: PropTypes.string,
   icon: PropTypes.string.isRequired,
   styles: PropTypes.object,
+  startTimeLocal: PropTypes.string.isRequired,
+  endTimeLocal: PropTypes.string.isRequired,
```

Comment:
`startTime` and `endTime` are no longer used so I would either:

- remove them from `propTypes` as well or
- keep using the original `startTime` and `endTime` props since AFAICT nothing changed inside this component besides the removal of the serialize+toMoment calls. BTW, the only reason these calls exist is to strip leading zeroes from hours before 10am, and that logic should probably be preserved since it's more human-friendly

Reply:
You are right! I believed startTime and endTime were still being used as part of posting the availability to the backend, but then I noticed the slot has its own action already associated for that purpose. I deleted `startTimeLocal` and `endTimeLocal`, and went back to `startTime` and `endTime`